### PR TITLE
DOC: update SciPy Roadmap for 1.0 release and recent discussions.

### PR DESCRIPTION
**Pull request by rgommers**

### CODE REVIEW

File: doc/ROADMAP.rst.txt [hunk 1]
```diff
@@ -69,33 +56,28 @@ a few missing or incomplete tutorials - this should be fixed.
...
   - Bento development has stopped, so will remain having an experimental,
     use-at-your-own-risk status.  Only the people that use it will be
     responsible for keeping the Bento build updated.
 
-A more complete continuous integration setup is needed; at the moment we often
-find out right before a release that there are issues on some less-often used
-platform or Python version.  At least needed are Windows (MSVC and MingwPy),
-Linux and OS X builds, coverage of the lowest and highest Python and NumPy
-versions that are supported.
+Continuous integration is in good shape, it covers Windows, macOS and Windows, as well
```
Comment (perimosocordiae) [inline doc/ROADMAP.rst.txt]: Windows twice

File: doc/ROADMAP.rst.txt [hunk 2]
```diff
@@ -258,10 +230,10 @@ things are done in `interpolate`), and eliminate any duplication.
 
 *Filter design*: merge `firwin` and `firwin2` so `firwin2` can be removed.
 
-*Continuous-Time Linear Systems*: remove `lsim2`, `impulse2`, `step2`.  Make
-`lsim`, `impulse` and `step` "just work" for any input system.  Improve
-performance of ltisys (less internal transformations between different
-representations). Fill gaps in lti system conversion functions.
+*Continuous-Time Linear Systems*: remove `lsim2`, `impulse2`, `step2`.  The
+`lsim`, `impulse` and `step` functions now "just work" for any input system.
+Further improve the performance of ``ltisys`` (less internal transformations
```
Comment (perimosocordiae) [inline doc/ROADMAP.rst.txt]: less -> fewer

**Review by perimosocordiae — APPROVED** [review]
A few minor tweaks, but this looks good.

File: doc/ROADMAP.rst.txt (hunk 1 above)
Comment (rgommers) [reply doc/ROADMAP.rst.txt]: fixed, thanks

File: doc/ROADMAP.rst.txt (hunk 2 above)
Comment (rgommers) [reply doc/ROADMAP.rst.txt]: yep, also fixed

File: doc/ROADMAP.rst.txt (hunk 1 above)
Comment (perimosocordiae) [reply doc/ROADMAP.rst.txt]: I still see two "Windows".

Comment (rgommers) [reply doc/ROADMAP.rst.txt]: sorry fixed now. missed ``-a`` from ``git commit -a --amend``.

**Comment by pv** [comment]
Thanks, let's get this in.

**Pull request merged**